siriuspy/siriuspy/devices/psconv.py

```diff
@@ -202,7 +202,8 @@ def _get_dev_if_dipole(devname):
     def _get_dev_if_trim(devname, proptype):
         if StrengthConv._is_trim(devname):
             # trims need dipole and family connectors
-            dev_dip = PSProperty('SI-Fam:PS-B1B2-1', 'Energy' + proptype)
+            dev_dip = PSProperty(
+                'SI-Fam:PS-B1B2-1', 'Energy' + proptype)
             devname = devname.replace(devname.sub, 'Fam')
```

Comment:
Use the method `SiriusPVName.substitute` here.

Reply:
`PSProperty` requires two arguments of type `str`: the first is a _devname_, the second is a property name. Are you suggesting changing `PSProperty` constructor so that `SiriusPVName.substitute` could be used to construct a unique constructor argument?

Reply:
Line 207.
```
devname = devname.replace(devname.sub, 'Fam')
```

Reply:
right under my nose! I will change!

Reply:
done, @fernandohds564